src/reason_parser.mly

```diff
@@ -2013,16 +2007,12 @@ parent_binder:
 
 value:
 /* TODO: factorize these rules (also with method): */
-    override_flag MUTABLE VIRTUAL as_loc(label) COLON core_type {
-      if $1 = Override then not_expecting 7 "not expecting equal - cannot specify value for virtual val"
-      else $4, Mutable, Cfk_virtual $6
-    }
```

Comment:
This case looks weird. Should equal sign be prohibited for any virtual field? If so, then the next case handles that.